chapters/io/index.rst

```diff
@@ -0,0 +1,315 @@
+How to handle input and output for Flows
+=======================================
+
+This chapter explains how to provide input data to the `Flow API`. Moreover, it describes how to handle the output via
+callback functions.
+
+Input
+-----
+The input data for the Flow operations `Flow.index`, `Flow.update` and `Flow.search` can be provided in three different ways.
+In the following, the input functionality is shown using the `Flow.index` function. `Flow.update` and `Flow.search` are used analogous.
+
+#. A single Document can be sent through the Flow as shown below.
+
+    .. highlight:: python
+    .. code-block:: python
+
+        with f:
+            f.index(Document('doc content'))
+
+#. An Iterable of Documents can be provided instead. The following example shows a Generator fed into a Flow.
```

Comment:
```suggestion
#. An Iterable of Documents can be provided instead. The following example shows a `Generator` fed into a `Flow`.
```

I guess these words should all be in backticks, as they are class / type names

Reply:
Thanks, done